Pages/PostMachineUI/PostMachineMain.razor

```diff
@@ -169,21 +169,14 @@ else if (placePackage)
     {
         ThisPostMachine = await PostMachineRepository.PostMachines.Include(p => p.ShipmentsWithThisSource)
             .Include(p => p.ShipmentsWithThisDestination).FirstAsync(p => p.Id == Id); //.GetAsync(Id);
+        loaded = true;
+
         ShipmentsWithThisSource = ThisPostMachine.ShipmentsWithThisSource;
         ShipmentsWithThisDestination = ThisPostMachine.ShipmentsWithThisDestination;
-        if (ShipmentsWithThisSource == null)
```

Comment:
Yeah, thats not happening since I specifically wrote `public ICollection<Shipment> ShipmentsWithThisSource { get; set; } = new List<Shipment>();` in Entity PostMachine.